source/companies/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Companies and Open Source
+url: /companies/
+tags: ["companies", "business", "navigation"]
+---
+
+# Companies and Open Source
+
+All modern digital infrastructure is dependent on open source software.
+Companies must think strategically about how they will engage with the
+open source projects on which they rely in order to ensure
+sustainability, and influence the direction of these projects for the
+benefit of their customers.
+
+## [Benefits to Companies](/companies/benefits.html)
+
+Active participation in open source projects provides significant strategic and operational benefits to companies, from talent acquisition to technology innovation and market positioning.
```

Comment:
```suggestion
Active participation in open source projects provides significant strategic and operational benefits to companies, from talent acquisition to technology innovation and market positioning.

Such cooperation works in both directions - when maintainers see company that is actively participating in the project, they are eager to help and solve problems that individuals from the company are raising.

Personal relationships with maintainers also makes it possible to save even weeks of going in a wrong direction in your deployments, because maintainers will know your context and will be able to help to make better decisions.
```

Reply:
> This is covered in the sub-page. The _index.md page is intentionally as terse as possible, with expansion on the sub-pages.

Sure I can suggest it in the sub-page